app/containers/Bundle.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
What exactly is this `Bundle` container necessary? Some inline comments explaining why this is necessary would be nice